kore/test/Test/Kore/Builtin/Set.hs

```diff
@@ -232,17 +236,56 @@ test_inElement =
 test_inElementSymbolic :: TestTree
 test_inElementSymbolic =
     testPropertyWithSolver
-        "in{}(x, element{}(x)) === \\dv{Bool{}}(\"true\")"
+        "in{}(x, element{}(x))\
+        \ === and{Bool{}()}(\\dv{Bool{}}(\"true\"), \\top{Bool{}}())"
         (do
-            key <- forAll (standaloneGen $ elementVariableGen intSort)
-            let patKey = mkElemVar key
-                patElement = mkApplySymbol elementSetSymbol [ patKey ]
-                patIn = mkApplySymbol inSetSymbol [ patKey, patElement ]
+            patKey <- forAll genKey
+            let patElement = mkApplySymbol elementSetSymbolTestSort [ patKey ]
+                patIn = mkApplySymbol inSetSymbolTestSort [ patKey, patElement ]
                 patTrue = Test.Bool.asInternal True
-                predicate = mkEquals_ patIn patTrue
-            (===) (Test.Bool.asPattern True) =<< evaluateT patIn
-            (===) Pattern.top                =<< evaluateT predicate
+                conditionTerm = mkAnd patTrue (mkCeil_ patElement)
+            actual <- evaluateT patIn
+            expected <- evaluateT conditionTerm
+            actual === expected
         )
+  where
+    -- genKeys :: Gen [TermLike Variable]
```

Comment:
Do you still need this and the comment a few lines below?

Reply:
These changes are outdated